src/Common/Help_Hub/Provider.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Provider for the Help Hub.
+ *
+ * @since   TBD
+ * @package TEC\Common\Help_Hub
+ */
+
+namespace TEC\Common\Help_Hub;
+
+use TEC\Common\Configuration\Configuration;
+use TEC\Common\Contracts\Service_Provider;
+
+/**
+ * Class Provider
+ *
+ * Registers the Help Hub logic and dependencies.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Help_Hub
+ */
+class Provider extends Service_Provider {
```

Comment:
Turn this into a Controller.

Reply:
I refactored a significant portion of the Help Hub code to better support future flexibility. With the goal of eventually using the Help Hub for both The Events Calendar (TEC) and Event Tickets (ET), I wanted to make the 'Resource' data dynamic.

I’ve set it up locally so that TEC and ET now have separate Help Hub pages, and everything functions exactly as intended. The Help_Hub_Factory currently manages the Resource section, allowing the creation of Help Hubs for specific types. While only tec_events is supported at the moment, enabling ET is as straightforward as passing event_tickets and creating a corresponding Resource Data class, similar to TEC_Hub_Resource_Data. This approach provides flexibility for future enhancements.

Here is a screenshot locally of separate Help Hubs for TEC/ET
![image](https://github.com/user-attachments/assets/33ea737d-5b8e-4d72-901b-f7e2594e675f)


With that in mind, should I still move my Provider to a Controller?